scripts/babel-relay-plugin/src/__fixtures__/fragmentRequiredArgs.fixture

```diff
@@ -0,0 +1,17 @@
+Input:
+var Relay = require('react-relay');
+var x = Relay.QL`
+  fragment on Viewer {
+    actorByName() {
```

Comment:
Is this meant as a test case that forgetting arguments without providing `pattern: true` still fails? If so, it is not actually testing that. This is failing because it is syntactically invalid GraphQL that does not *parse*. If you correct it to the syntactically correct `actorByName { name }`, it *will* parse and will also pass validation because of the removed "ProvidedNonNull" validation rule above.

We need to keep the non-null validation rule for queries/fragments.